src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -0,0 +1,21 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+
+<Project>
+ 
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
+  
+  <Target Name="RunPackageValidation" AfterTargets="Pack">
+    <PropertyGroup>
+      <PackagePath Condition="'$(PackagePath)' == ''">$(PackageOutputPath)$(PackageID).$(Version).nupkg</PackagePath>
+      <RunApiCompat Condition="'$(RunApiCompat)' == ''">false</RunApiCompat>
+      <ValidateWithPreviousVersion Condition="'$(ValidateWithPreviousVersion)' == ''">false</ValidateWithPreviousVersion>
+    </PropertyGroup>
+
+    <ValidatePackage PackagePath="$(PackagePath)"
+                     RuntimeGraph="$(RuntimeGraph)"
```

Comment:
What is the RuntimeGraph? Is that the same thing we use in dotnet/runtime and if yes, don't we want to remove it and not support it at all?

Reply:
Got confused with the packageIndex. Sorry, disregard.